.github/workflows-source/ci-main.yml

```diff
@@ -4,14 +4,13 @@ on:
   merge_group:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-  schedule: # Todo: switch to push on master and remove schedule once we have more macos runners
-    - cron: "0 * * * *"
-  pull_request:
-  # Used as reusable workflow within release-testing workflow
-  workflow_call:
   push:
     branches:
+      - master
       - 'dev-gh-*'
+  pull_request:
```

Comment:
Is this still run on pull requests?

_edit:_ ok I just realized this workflow contains both the namespace & intel jobs. Why not split it into two, since they have different triggers?

Reply:
The main benefit in keeping them together is the release-testing can trigger the entire ci-main workflow.